docs/data/joy/components/sheet/SheetUsage.js

```diff
@@ -0,0 +1,244 @@
+import * as React from 'react';
```

Comment:
I created a reusable component for showing the usage playground. Please take a look at `ChipUsage.js` or `SliderUsage.js` for how to use.

Reply:
Sure, done!